src/test/java/plugins/AntPluginTest.java

```diff
@@ -23,15 +32,46 @@ public class AntPluginTest extends AbstractJUnitTest {
     private static final String INSTALL_VERSION_1_10 = "1.10.5";
     private static final String INSTALL_NAME_1_10 = "ant_" + INSTALL_VERSION_1_10;
 
+    private static final String NATIVE_ANT_NAME = "native_ant";
+
+    private static final String BUILD_FILE = "custom-build-file.xml";
+
     FreeStyleJob job;
+    private AntBuildStep step;
+
+    @Inject private DockerContainerHolder<SshAgentContainer> docker;
+
+    private SshAgentContainer sshd;
+    private DumbSlave agent;
 
     @Before
     public void setUp() {
         job = jenkins.jobs.create(FreeStyleJob.class);
```

Comment:
this is creating a Freestyle job even for the jobs that tests.
would make the tests quicker if this (and the probably `FreeStyle job filed) was just inlined in the tests that need it.